### PR TITLE
Create_Build_Your_Own_Adventure.md

### PR DESCRIPTION
**Pull request by necode2**
Thanks for making a PR! Please fill in the below:

Name: Noor Elbanna

Do you run a club? 
Yes! I run a coding club at my school with three other leaders!

What relevant experiences do you have to help you write this Jam?
I've been programming since the 2nd grade and now as a sophomore in high school building adventures games have been one of my favorites! I've been programming in several languages, mainly Python, however I have experience with Java, HTML and CSS. This Jam would also provide me with the opportunity of combining things I have learned over the years into one package. 

Will you be available on Slack to provide support to Hack Clubbers who run into issues while going through your Jam?
Yes! I will be active and even try to provide fun challenges alongside the project.

### CODE REVIEW

File: jams/Build_Your_Own_Adventure.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+### Platform Limitations: 
+Browser-based Replit Development, so available on all platforms (+ App available on both iOS and Android)
+
+### Time estimation: 
+45 Minutes (Depends on how detailed the jammer wants their build to be)
+
+## Breakdown
+Set up: Fork either the Java or Python replit starter with the necessary materials
+### Part 1: (ETC: Simplest and can be completed in under 30 minutes)
```
Comment (zeyuyaoy) [inline jams/Build_Your_Own_Adventure.md:31]: This part should be setting up the Replit project and installing relevant dependencies if needed

File: jams/Build_Your_Own_Adventure.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Follow the instructions and start building a text-based Java or Python Adventure Game
+          - Plan out your game by creating characters, storyline, and options
+
+Success!!! You now have a working text-based Choose your own adventure game! 
+
+### Part 2: (ETC: Increased in difficulty and can take up to 1 hour)
```
Comment (zeyuyaoy) [inline jams/Build_Your_Own_Adventure.md:37]: As per my comment above, you should shift all these sections down - additionally, I think. that part 1 and 2 should be the sections with the most details, while allowing clubs themselves to explore how they want to take this

**Review by zeyuyaoy — CHANGES_REQUESTED** [review]
Can you answer the questions found at the bottom of the [example file?](https://github.com/hackclub/jams/blob/main/jams/example.md)

**Review by linkai101 — CHANGES_REQUESTED** [review]
Great idea! I loved making text adventure games when I started learning to code.

Just a few thoughts:
1) I assume the 3 parts are supposed to be done in separate meetings, which might not be the most appealing to most club members. IMO the project would be dragged on for too long to be held for 3 club meetings, especially one meeting just for adding music.

I could imagine a meeting go like this:
Step 1--Creating idea/storyline of adventure (could be done on paper/notes and discussion)
Step 2--Implementing the game with code: code setup, brief codealong (maybe advanced jammers could use UI while beginners could use text-based)
...

2) The breakdown, especially for part 1, is super vague. Could you add more specific steps and time durations? (e.g. overview of user input and if/else statements, etc.) Mainly steps for actually implementing the game with code.

I'm interested in hearing what others have to say, but I like the idea! Hope all of this makes sense.

**Comment by sarthaktexas** [comment]
I would love to merge this, can you finish the changes request above @necode2 ?

**Comment by necode2** [comment]
Thank you so much for all the feedback! I'll implement this and have a new version ready tomorrow! Also, would it be possible to get a quick snapshot of the questions from the example file? I am unable to open it from my end.

**Comment by sahitid** [comment]
love it! lgtm

**Pull request merged**